core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfigService.java

```diff
@@ -215,4 +217,40 @@ String getExperimentFeatureKey(String experimentId) {
 
         return featureVariableKeyMap;
     }
+
+    @VisibleForTesting
+    Map<String, OptimizelyAttribute> getAttributesMap(List<Attribute> attributes) {
```

Comment:
We do not use this attributesMap and eventsMap in OptimizelyConfig. I think we can delete these.

Reply:
removed